src/lib/core/style/_menu-common.scss

```diff
@@ -0,0 +1,45 @@
+@import './variables';
+@import './elevation';
+@import './list-common';
+
```

Comment:
Add top-level comment here saying that these styles are shared between menu and select.